tor_ocr/core/config.py

```diff
@@ -63,19 +67,23 @@ class Config(object):
     """
 
     # API key for later overwriting based on contents of filesystem
-    bugsnag_api_key = None
+    bugsnag_api_key: str = None
 
-    debug_mode = False
+    debug_mode: bool = False
 
     # Global flag to enable/disable placing the triggers
     # for the OCR bot
-    OCR = True
+    OCR: bool = True
+    ocr_delay: int = None
 
     # Name of the bot
-    name = None
-    bot_version = '0.0.0'  # this should get overwritten by the bot process
+    name: str = None
+    bot_version: str = '0.0.0'  # this should get overwritten by the bot process
```

Comment:
Why is there a version string here and also in `main.py`?

Reply:
Old compatibility thing that can be cleaned up, just not in the scope of this PR